store/sqlstore/upgrade.go

```diff
@@ -1438,6 +1438,9 @@ func upgradeDatabaseToVersion640(sqlStore *SqlStore) {
 		sqlStore.GetMaster().ExecNoTimeout("ALTER TABLE Users DROP COLUMN AcceptedTermsOfServiceId")
 	}
 
+	sqlStore.CreateColumnIfNotExistsNoDefault("OAuthApps", "Scopes", "Text", "VARCHAR(1024)")
```

Comment:
@isacikgoz @larkox `CreateColumnIfNotExistsNoDefault` was removed as unused in https://github.com/mattermost/mattermost-server/pull/19354. Any recommendation on how to resolve the conflict?

Reply:
The new migration system has to be used, so these have to be removed and handled in the files in `/db/migrations/`. There should be some documentation about how to handle this, but I don't remember where.

Reply:
https://github.com/mattermost/mattermost-server/pull/19214 can be used an an example